server/CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ else()
 	else()
 		add_executable(vcmiserver ${server_SRCS} ${server_HEADERS})
 	endif()
+	set(BINARY_TARGETS ${BINARY_TARGETS} vcmiserver PARENT_SCOPE)
```

Comment:
Here you ignore the if branch, but in other cases where static library is also involved you add target to the variable unconditionally. Why?

Reply:
on the if branch there is SHARED library so it is also a binary

Reply:
I mean the parent `if`, line 53